composites/OnboardingWizard/config/production-config.js

```diff
@@ -11,25 +11,25 @@ let configuration = {
 	},
 	"fields": {
 		"upsellConfigurationService": {
-			"componentName": "HTML",
+			"component": "HTML",
```

Comment:
Yup, the wordpress API already uses `component`. If this is changed the wizard does not work in WP anymore.